rancher/dashboard
# Apply theme colours to branding text by default. #4501

## Summary

Issue #4501 - Branding Config: Text color shown as white when not configured

## Technical notes
- Updated BannerSettings component to retrieve the default values using `getComputedStyle`.
- `ColorInput` component would use the `:default-value` if `:value === null`
- Same default `--banner-text-color` should apply to Header, Footer and Banner consent.

## Steps to test

1. Default settings
    1. Navigate to Configuration > Global Settings > Branding
    2. Tick ALL or each of the `Show banner in header`, `Show banner in footer` and `Show Consent Banner on Login Screen`
    3. Light theme - `Text Color` option and should highlight `#141419`
    4. Dark theme - `Text Color` option and should highlight `#B6B6C2`
    5. Applying the setting should ensure selected colour is set
    6. Colour should remain in default of the theme (when switching themes), if user didn't change the colour and only edit the text.
    7. 3 Banners `Header`,`Footer` and `Banner Consent` text colours are set independently.

2. Legacy to Default settings
    1. If have legacy UI navigate to Settings > ui-banner.
    2. Update the banner settings json to be `{}`
    3. Test again with the 1. Default settings steps. UI Should work the same way
